src/app/routes/getIndexData/index.js

```diff
@@ -0,0 +1,50 @@
+import 'isomorphic-fetch';
+import nodeLogger from '../../helpers/logger.node';
+import getBaseUrl from '../helpers/getBaseUrl';
+
+const logger = nodeLogger(__filename);
+
+const upstreamStatusCodesToPropagate = [200, 404];
+
+const getIndexData = async ({ match }) => {
+  const { service } = match.params;
+  let data;
+  let status;
+  let baseUrl = process.env.SIMORGH_BASE_URL;
+
+  if (
+    typeof window !== 'undefined' &&
+    window.location &&
+    window.location.origin
+  ) {
+    baseUrl = getBaseUrl(window.location.origin);
+  }
+
+  const url = `${baseUrl}/${service}/index/front_page.json`;
```

Comment:
An alternative to duplicating 98% of the original `getInitialData` method is to keep that and introduce a page type prop allowing us to construct the url based on the page type.